Dockerfile

```diff
@@ -21,3 +21,7 @@ RUN /usr/local/bin/plugins.sh /usr/share/jenkins/ref/active.txt
 # copy job definitions
 RUN mkdir /usr/share/jenkins/ref/jobs
 COPY jobs/* /usr/share/jenkins/ref/jobs/
+
+# copy jenkins-launchpad-plugin config
+RUN mkdir /var/jenkins_home/.jlp
```

Comment:
It seems that this directory gets wiped on jenkins expansion, if we copy it to /usr/share/jenkins/ref it would be persisted:

RUN mkdir /usr/share/jenkins/ref/.jlp
COPY jlp.config /usr/share/jenkins/ref/.jlp/

Thanks!